.moon/toolchain.yml

```diff
@@ -25,7 +25,7 @@ node:
   aliasPackageNames: "name-and-scope"
 
   # Dedupe dependencies after the lockfile has changed.
-  dedupeOnLockfileChange: true
```

Comment:
There is a bug in yarn dedupe that doesn't understand our `nohoist` settings for yarn workspaces. 

This disabled `yarn dedupe` which isn't the worst thing in the world